suite-native/app/e2e/fixtures/walletWithBtcAcc.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
Please explain why is this necessary for future reference or remove if it's not needed 🙏

Reply:
Oh, I forgot to mention it..  the issue is with the `types` of those parameters I added. Initially, they were already part of @jbazant ’s PR, but since they didn’t pass the type check, I assumed they might have been removed. However, they are actually required and without them, the trading buy flow fails.

My guess is that although the values don’t satisfy the types, this is the bare minimum required for the address to be displayed in the flow.